docs/examples.rst

```diff
@@ -185,9 +185,15 @@ If you only want a few specific fields, save time by asking for them explicitly:
     issue = jira.issue('JRA-1330', fields='summary,comment')
 
 Reassign an issue::
+    Reassigning an issue can be done either by providing the username/accountId (hosted/cloud) or the User Resource object itself::
```

Comment:
![image](https://github.com/user-attachments/assets/33f7e0e3-5fae-4075-bb76-a2753878c7f0)

Remove the `::` after `Reassign an issue`

and also there should be no indentation for the first line, and therefore codeblock underneath also should get indented back

```suggestion
Reassigning an issue can be done either by providing the username/accountId (hosted/cloud) or the User Resource object itself::
```